### ISSUE TITLE
Italic on html/jsx tag attributes

### ISSUE THREAD

**Issue by kud** [saadq/Materialize#85]
I use the font `Operator Mono` and I'd love to have my html/jsx tag attributes in italic. Could it be possible to add it?

**Comment by kud** [comment]
In the meantime: https://github.com/SintrumIT/oceanic-next-italic

**Comment by saadq** [comment]
For Materialize, the main idea behind this package is making Material versions of UI themes that match popular color schemes. I added in color schemes bundled with the package, but I did that for simplicity and don't really plan to make adjustments to color schemes. It is recommended that for changes like this you just head over to the color scheme you want and ask for a version/changes there.

Also, I am happy to help anyone who would like some assistance in modifying their personal color scheme file if they ask, but most likely won't make any changes to the color schemes in the repo – Again, the focus of this package is the UI themes. Hope you understand!
